custom/icds_reports/management/commands/sql_scripts/nos_of_deaths.sql

```diff
@@ -9,9 +9,13 @@ FROM (
       SELECT awc_location.state_name AS state,
              awc_location.district_name AS district,
              awc_location.awc_name AS awc,
-             date_trunc('month',person_case.date_death) as death_month
+             date_trunc('month', person_case.date_death) as death_month
       FROM "%(person_table_name)s" AS person_case
-      INNER JOIN "%(awc_location_table_name)s" AS awc_location
+      INNER JOIN awc_location
```

Comment:
if you want all the locations with no deaths to be included, you will need to make this a `RIGHT JOIN`. you will need to add another table, perhaps `icds_months` if you want all months to be included for each location

Reply:
for month, putting all months is not necessary.
If I will right join with awc_location wont it also include the un_launched awcs also which we dont want. 
How I thought it, I can just export the list of launched awcs and make the UNION of to dataframes separately.

Reply:
It would have the same filter as the rest of the query because of the `WHERE` clause below (which i believe includes awcs that have not been launched, but excludes test states).

If you do not need all months that should be fine. I think this includes all months where there is a death